response/page_path_test.go

```diff
@@ -0,0 +1,265 @@
+package response
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type TestCase struct {
+	uri      string
+	expected string
+}
+
+func TestGetPagePath(t *testing.T) {
+	Convey("Given a list of URIs", t, func() {
+		ctx := context.Background()
+
+		visualisationsEndpoints := []TestCase{
+			{
+				uri:      "/visualisations/dvc1945/seasonalflu/index.html",
+				expected: "/visualisations/dvc1945",
+			},
+			{
+				uri:      "/visualisations/dvc1945/duetoinvolving/fallback.png",
+				expected: "/visualisations/dvc1945",
+			},
+		}
+		resourceEndPointPaths := []TestCase{
```

Comment:
We're missing some real examples of other endpoints, like `/embed` or `/chart` (and others). We should ask someone at ONS, so that they can tell us how to find the missing ones, and add them here.